authors.py

```diff
@@ -63,7 +63,7 @@ def connect_author_to_feide(connect_author, connect_scn, connect_payload):
 if not USER_POOL_ID:
     quit('Set environment variable AWS_USER_POOL_ID to correct User Pool Id')
 
-CLIENT_ID = os.environ['AWS_CLIENT_ID']
+CLIENT_ID = os.environ['AWS_USER_POOL_WEB_CLIENT_ID']
 if not USER_POOL_ID:

```

Comment:
Shoul this be CLIENT_ID and not USER_POOL_ID ?

Reply:
Ups